pyscriptjs/tests/integration/support.py

```diff
@@ -126,6 +126,17 @@ def init_page(self, page):
         page.on("console", self._on_console)
         page.on("pageerror", self._on_pageerror)
 
+    def run_js(self, code):
```

Comment:
docstring, please!
And also a test in `test_00_support.py`

Reply:
done!